### ISSUE TITLE
Unknown inspection suppression false-positive

### ISSUE THREAD

**Issue by 3DFace** [kalessil/phpinspectionsea#151]
For example `SpellCheckingInspection`:

Start with
```
   $x = 'asdzxc';
```
SpellChecking inspection will detect typo in string literal. Suppress it (ALT+Enter ... suppress for statement)
```
   /** @noinspection SpellCheckingInspection */
   $x = 'asdzxc';
```

Now `SpellCheckingInspection` seems unknown to "Unknown inspection suppression".

**Comment by funivan** [comment]
`SpellCheckingInspection` this is standard inspection of the phpstorm. So you need to write issue in https://youtrack.jetbrains.com/issues/WI

**Comment by kalessil** [comment]
Actually it's possible that the flaw on our side - I'll need to check where the inspections is located first.

**Comment by 3DFace** [comment]
@funivan Right, SpellCheckingInspection is standard inspection. And it works fine.
But UnknownInspectionInspection didn't recognize it. But it should, I think.

**Comment by kalessil** [comment]
Fixed, the issue is in between IDEs and the EA plugin - such things doesn't happen often.
